x/liquidfarming/spec/07_hooks.md

```diff
@@ -0,0 +1,31 @@
+<!-- order: 7 -->
+
+# Hooks
+
+The `Liquidfarming` module has the events triggered by the hooks from `farming` module.
+The `Liquidfarming` module uses the following hooks registered in `farming` module.
```

Comment:
We can remove the first sentence as it sounds duplicate.
```suggestion
The `Liquidfarming` module uses the following hooks registered in `farming` module.
```

Reply:
Reflected.